cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2754,6 +2754,83 @@ action.cyences_send_email_action = 1
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Google Workspace
+
+
+[Email - With Known Abuse Web Service Link]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 57 * * * *
+description = This alert will trigger mail contains Known Abuse Web Service Link. \
+NOTE: cs_email_user_domain macro must be configured with the domain name in order to use this alert. \
+Data Collection - Gmail data (Splunk Add-on for Google Workspace). 
+dispatch.earliest_time = -1d@h
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_gsuite` sourcetype="gws:gmail" _index_earliest=-61m@m _index_latest=-1m@m \
+| rename "message_info.link_domain{}" as link_domain, "message_info.is_spam" as is_spam, recipient as RecipientAddress, src_user as SenderAddress, subject as Subject \
+| search `cs_email_known_abuse_web_service_link` \
+| where not src_user_domain=`cs_email_user_domain` and recipient_domain=`cs_email_user_domain` \
+| stats count, min(_time) as firstTime, max(_time) as lastTime, values(link_domain) as link_domain by SenderAddress, RecipientAddress, Subject, is_spam \
+|  eval cyences_severity = "low" \
+| `cs_human_readable_time_format(firstTime)` \
+| `cs_human_readable_time_format(lastTime)` \
+| sort - count \
+| `cs_email_with_known_abuse_web_service_link_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_email_with_known_abuse_web_service_link_filter
+action.cyences_notable_event_action.contributing_events = `cs_gsuite` sourcetype="gws:gmail" | rename "message_info.link_domain{}" as link_domain, "message_info.is_spam" as is_spam, recipient as RecipientAddress, src_user as SenderAddress, subject as Subject | search `cs_email_known_abuse_web_service_link` | where not src_user_domain=`cs_email_user_domain` and recipient_domain=`cs_email_user_domain`
+action.cyences_notable_event_action.system_compromised_search = | stats count by SenderAddress, RecipientAddress, Subject, link_domain
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_gsuite` sourcetype="gws:gmail" | rename "message_info.link_domain{}" as link_domain, "message_info.is_spam" as is_spam, recipient as RecipientAddress, src_user as SenderAddress, subject as Subject | search `cs_email_known_abuse_web_service_link` | where not src_user_domain=`cs_email_user_domain` and recipient_domain=`cs_email_user_domain` SenderAddress=$row.SenderAddress$ RecipientAddress=$row.RecipientAddress$ Subject=$row.Subject$ link_domain=$row.link_domain$
```

Comment:
Such renames are good to be done from props.conf instead

Reply:
Is these renames really needed?

Reply:
I've kept these rename same as other email alerts. because in future, we might to add o365 support.

Reply:
Perfect